### PR TITLE
[5.3] Add On-Ramp Aggregator What's new modal

### PR DESCRIPTION
**Pull request by wachunei**
> ⚠️ This PR Depends on #4479 

This PR adds the content of the On Ramp Aggregator _What's new_ modal
- [x] Title
- [x] Description
- [x] Image
- [x] CTA Label

This PR also improves:
- Touchable area of pages is maximized
- Pages indicators are touchable
- Hides indicators and disable horizontal scrolling when there is only one (1) page
- Improves version typing to be correct

### CODE REVIEW

File: app/components/UI/WhatsNewModal/index.js [hunk 1]
```diff
@@ -51,7 +51,13 @@ const createStyles = (colors) =>
       flex: 1,
       width: slideItemWidth,
       paddingHorizontal: modalPadding,
+    },
+    slideItemContainerContent: {
       paddingBottom: 16,
+      flexGrow: 1,
+    },
+    slide: {
+      flex: 1,
```
Comment (wachunei) [inline app/components/UI/WhatsNewModal/index.js:60]: This allows the slide to be full height

File: app/components/UI/WhatsNewModal/index.js [hunk 2]
```diff
@@ -119,11 +125,11 @@ const createStyles = (colors) =>
   });
 
 const WhatsNewModal = (props) => {
+  const scrollViewRef = useRef();
```
Comment (wachunei) [inline app/components/UI/WhatsNewModal/index.js:128]: This ref is later used to scroll when indicators are pressed.

File: app/components/UI/WhatsNewModal/index.js [hunk 3]
```diff
@@ -236,9 +242,13 @@ const WhatsNewModal = (props) => {
...
     return (
-      <ScrollView key={key} style={styles.slideItemContainer}>
+      <ScrollView
+        key={key}
+        style={styles.slideItemContainer}
+        contentContainerStyle={styles.slideItemContainerContent}
+      >
         <TouchableWithoutFeedback>
-          <View>
+          <View style={styles.slide}>
```
Comment (wachunei) [inline app/components/UI/WhatsNewModal/index.js:251]: This allows slides to be full height

File: app/components/UI/WhatsNewModal/index.js [hunk 4]
```diff
@@ -280,9 +290,11 @@ const WhatsNewModal = (props) => {
               </TouchableOpacity>
             </View>
           </View>
-          {featuresToShow && (
+          {whatsNew.slides.length > 0 && (
```
Comment (wachunei) [inline app/components/UI/WhatsNewModal/index.js:293]: Conditionally rendering based on `featuresToShow` was causing a content flash when CTAs were pressed.

File: app/components/UI/WhatsNewModal/index.js [hunk 5]
```diff
@@ -292,17 +304,30 @@ const WhatsNewModal = (props) => {
...
+                      <View
+                        style={[
+                          styles.slideCircle,
+                          currentSlide === index && styles.slideSolidCircle,
+                        ]}
+                      />
+                    </TouchableWithoutFeedback>
+                  ))}
+                </View>
+              )}
```
Comment (wachunei) [inline app/components/UI/WhatsNewModal/index.js:330]: Indicators are shown only when there are more than 1 pages and they are pressable.

**Review by Cal-L — APPROVED** [review]
Made a small adjustment. LGTM

**Comment by wachunei** [comment]
Should I flip the `onlyUpdates` property 🤔 ?

**Review by Cal-L — COMMENTED** [review]
Forgot to submit review, left comment!

File: app/components/UI/WhatsNewModal/whatsNewList.ts [hunk 6]
```diff
@@ -5,7 +5,7 @@ import { WhatsNew } from './types';
 
 export const whatsNew: WhatsNew = {
   // All users that have <1.0.7 and are updating to >=1.0.7 should see
```
[the remark this answers is not in the document]
Comment (wachunei) [reply app/components/UI/WhatsNewModal/whatsNewList.ts]: Updated in https://github.com/MetaMask/metamask-mobile/pull/4484/commits/b63591fe156bb741cbbb298f417d8c34c032c867

**Review by Cal-L — APPROVED** [review]
LGTM

**Pull request merged**